requirements/base.in

```diff
@@ -1,3 +1,5 @@
+# pin arror to 0.14.4, because 0.15.0 doesn't like the 2018/04/2018-04-09-15-30-00 format
+arrow==0.14.4
```

Comment:
I wonder if we can ditch `arrow` at some point. I remember we needed it for some logic inside beetmover but we may be using some other package or so in the future. Moreover, lots of code will die with EOL of esr52 when we'll be having declarative artifacts everywhere. I'll track it under https://github.com/mozilla-releng/beetmoverscript/issues/253